storage/innobase/fil/fil0fil.cc

```diff
@@ -2084,6 +2093,22 @@ fil_rename_tablespace_check(
 
 	exists = false;
 	if (os_file_status(new_path, &exists, &ftype) && !exists) {
+		auto schema_path= fil_make_filepath(new_path, NULL, NO_EXT,
+						    true, true);
+		if (schema_path == NULL) {
+			return DB_ERROR;
+		}
+		exists= false;
+		if (os_file_status(schema_path, &exists, &ftype) && !exists)
+		{
```

Comment:
In the archaic-style formatted code, the `{` is supposed to be on the same line, and there is a space before `=`.

Reply:
right.